docs/streams/developer-guide.html

```diff
@@ -267,21 +295,532 @@
+	  the configured default serdes. </li>
+	  Several variants of <code>stream</code> exist to e.g. specify a regex pattern for input topics to read from.</td>
+      </tr>
+      <tr>
+        <td><b>Table</b>: input topic(s) -> <code>KTable</code></td>
+	<td>
+	  Reads the specified Kafka input topic into a <code>KTable</code>. The topic is interpreted as a changelog stream,
+	  where records with the same key are interpreted as UPSERT aka INSERT/UPDATE (when the record value is not <code>null</code>) or
+	  as DELETE (when the value is null) for that key. 
+	  <p>
+	  Slightly simplified, in the case of a KTable, the local KTable instance of every application instance will be populated
+	  with data from only a subset of the partitions of the input topic. Collectively, i.e. across all application instances, all
+	  the partitions of the input topic will be read and processed.
+	  <p>
+	  You must provide a name for the table (more precisely, for the internal state store that backs the table). This is required,
+	  among other things, for supporting interactive queries against the table.
+
+	  <pre class="brush: java;">
+        import org.apache.kafka.common.serialization.Serdes;
+        import org.apache.kafka.streams.kstream.KStreamBuilder;
+        import org.apache.kafka.streams.kstream.KTable;
+
+        KStreamBuilder builder = new KStreamBuilder();
+
+        KTable&lt;String, Long&gt; wordCounts = builder.table(
+	        Serdes.String(), /* key serde */
+	        Serdes.Long(),   /* value serde */
+	        "word-counts-input-topic", /* input topic */
+	        "word-counts-partitioned-store" /* table/store name */);
+	  </pre>
+
+	  When to provide serdes explicitly:
+	  <ul>
+	    <li>If you do not specify serdes explicitly, the default serdes from the configuration are used.</li>
+	    <li>You must specificy serdes explicitly if the key and/or value types of the records in the Kafka input topic do not
+	      match the configured default serdes.</li>
+	  </ul>
+	  
+	  Several variants of <code>table</code> exist to e.g. specify the <code>auto.offset.reset</code>
+	  policy to be used when reading from the input topic.
+	</td>
+      <tr>
+        <td><b>Global Table</b>: input topic -> <code>GlobalKTable</code></td>
+	<td>
+	  Reads the specified Kafka input topic into a <code>GlobalKTable</code>. The topic is interpreted as a changelog stream, where records
+	  with the same key are interpreted as UPSERT aka INSERT/UPDATE (when the record value is not <code>null</code>) or as DELETE (when the
+	  value is <code>null</code>) for that key. 
+	  <p>
+	    Slightly simplified, in the case of a GlobalKTable, the local GlobalKTable instance of every application instance will be
+	    populated with data from all the partitions of the input topic. In other words, when using a global table, every application
+	    instance will get its own, full copy of the topic’s data.
```

Comment:
The `’` in "topic's data" is rendered oddly in my browser.

Screenshot:
<img width="826" alt="screen shot 2017-07-17 at 17 10 47" src="https://user-images.githubusercontent.com/294849/28275013-e939402c-6b12-11e7-8ae2-20dc60e8b74e.png">